### DISCUSSION TITLE
Published app that used to run fine gets error message

### DISCUSSION THREAD

**Discussion by Kultursylt** [Budibase/budibase#9058] [Help] [answered]
Hi

I run a questioneer (https://assessment.fabel.se/)that used to run like a clock, it has about 50 steps and uses the and was launched during the summer.

Now I suddenly get connect ECONNREFUSED 52.232.2.252:5432 when running it. Sometimes it happens on the third step and sometimes later. In the backend, it says that it is running on the latest version. I tried to revert it to version 1.0.206 but had the same problem.

What could be the problem?

**Comment by andz-bb** [comment, marked as answer]
hey @Kultursylt - it looks like there could be a problem with the connection to your datasource. Are you using an external Postgres DB for this? The error message suggests that your app could be failing to connect to the DB server.

**Reply by Kultursylt** [reply]
Yes we are using an external Postgres database. Merde! It seem I have to find a lost password.
